foundry/applications/cameraViews.d.ts

```diff
@@ -1 +1,180 @@
-declare class CameraViews extends Application {}
+/**
+ * The Camera UI View that displays all the camera feeds as individual video elements.
+ */
+declare class CameraViews extends Application {
+  /**
+   * @param webrtc - The WebRTC Implementation to display
```

Comment:
maybe file a bug with foundryvtt about this unused parameters? I suppose it's just legacy.

Reply:
I filed an issue for it. https://gitlab.com/foundrynet/foundryvtt/-/issues/4762